src/pages/RegisterPage.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, PasswordInput, TextInput } from "@mantine/core";
+import { useState } from "react";
+import { CreateUserDto } from "../types/dto";
+import createUser from "../api/users";
+
+function RegisterPage() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (username.trim().length == 0 || password.trim().length == 0) return;
+
+    const createUserDto: CreateUserDto = {
+      username: username.trim(),
```

Comment:
maybe good opportunity to reduce code duplication

refactor to 
const usernameTrimmed = usernmae.trim()

and same for password

Reply:
done